code/modules/mob/living/carbon/human/species/station/diona/diona.dm

```diff
@@ -23,7 +23,8 @@
 	siemens_coefficient = 0.3
 	eyes = "blank_eyes"
 	show_ssd = "completely quiescent"
-	num_alternate_languages = 1
+	num_alternate_languages = 2
+	secondary_langs = list(LANGUAGE_ROOTSONG, LANGUAGE_NRAL_SONG)
```

Comment:
You can remove LANGUAGE_ROOTSONG from here, all dionae start with it already.

Reply:
thanks alb